src/Sylius/Bundle/ApiBundle/Resources/config/routing.yml

```
@@ -10,6 +10,12 @@ sylius_api_admin_authentication_token:
     path: /admin/authentication-token
     methods: ['POST']
 
+sylius_api_admin_sales_statistics:
+    path: /admin/sales-statistics
+    methods: ['GET']
+    defaults:
+        _controller: Sylius\Bundle\ApiBundle\Controller\GetSalesStatisticsAction
+
```

```suggestion
sylius_api_admin_sales_statistics:
    path: /%sylius.security.new_api_route%/sales-statistics
    methods: ['GET']
    defaults:
        _controller: Sylius\Bundle\ApiBundle\Controller\GetSalesStatisticsAction

```

Idk if we can use a parameter that way, but from what I remember it's possible in general.

---

Don't think this is valid, IIRC the whole file is prefixed

---

You're right, I thought it would be `/admin/`, but in fact it'd be `/api/v2/admin/sales-statistics